.github/workflows/fuzz.yml

```diff
@@ -0,0 +1,19 @@
+name: Fuzz
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install
+        run: |
+          python -m pip install -U pip
```

Comment:
Interesting there's a Python available without `actions/setup-python`, looks like it has Python 3.8.

In any case, shall we include an explicit `actions/setup-python` so we don't get any surprises when the version changes? Can we use 3.10?

Reply:
Ubuntu ships a Python with it - that's the one that's there by default. It'll get upgraded whenever a new `ubuntu` image is added to GitHub Actions and `ubuntu-latest` shifts. It's a complete Python3+pip environment and this test shouldn't be Python version dependent so I'd be happy to use it as is.